### DISCUSSION TITLE
Add installation method (manifest) to Scoop Installer

### DISCUSSION THREAD

**Discussion by pandaero** [devcompl/TTController#162] [Idea]
For easier installation / app management a well-made manifest for [ScoopInstaller](https://github.com/ScoopInstaller) would be great

**Comment by pandaero** [comment]
probably under the [Versions](https://github.com/ScoopInstaller/Versions) bucket

**Comment by MoshiMoshi0** [comment]
What would be the benefit here? There isnt really anything to install, just download a zip and run the exe.
Also why not chocolatey?
